packages/repository-tests/src/crud/transactions.suite.ts

```diff
@@ -169,7 +174,20 @@ export function transactionSuite(
             {},
           );
           expect(toJSON(created)).to.deepEqual(toJSON(foundOutsideTransaction));
+
+          setTxToUndefined(tx);
         });
+
+        // temporary workaround for "Release called on client which has
+        // already been released to the pool." for PostgreSQL until
+        // tx.isActive is available
+        //FIXME: remove this once tx.isActive becomes available
+        // see https://github.com/strongloop/loopback-next/issues/3471
+        function setTxToUndefined(t: Transaction | undefined) {
+          if (ds.connector && ds.connector.name === 'postgresql') {
+            t = undefined;
```

Comment:
This is not working correctly as `t = undefined` inside the function won't change the value of `tx` argument from the caller. You can return `undefined` or `t` conditionally and use it  as `tx = setTxToUndefined(tx)`.

Reply:
FYI: this can be further simplified as:
```ts
return ds.connector && ds.connector.name === 'postgresql' ? undefined: t;
```